pcsx2/gui/MainFrame.cpp

```diff
@@ -53,6 +53,26 @@ wxMenu* MainEmuFrame::MakeStatesSubMenu( int baseid, int loadBackupId ) const
 	return mnuSubstates;
 }
 
+void MainEmuFrame::UpdateStatusBar()
+{
+	wxString temp(wxEmptyString);
+
+	if (g_Conf->EnableFastBoot)
+		temp += "Fast Boot - ";
+
+	if (g_Conf->CdvdSource == CDVD_SourceType::Iso)
+		temp += "Selected: '" + wxFileName(g_Conf->CurrentIso).GetFullName() +"' ";
+
+#ifdef __M_X86_64
+	temp+= "(64 bit) ";
+#else
+	temp += "(32 bit) ";
+#endif
```

Comment:
I think it would be better for 32/64bit info to be moved in it's own bar/column like CDVD selected boot type. Maybe move it at the end and also shrink the cdvd one.

Currently if the file name is too big build info gets cut.

Reply:
You might see how it looks now. Getting it not to cut off "NoDisc" when resizing those is a bit tricky, too.